src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+public enum LottoRank {
+    FIRST(6, false, 2000000000),
```

Comment:
가독성을 위해 언더 스코어를 활용해 보셔도 좋을 것 같아요
```suggestion
    FIRST(6, false, 2_000_000_000),
```